client/src/main/java/io/pravega/client/netty/impl/ClientConnection.java

```diff
@@ -37,7 +38,16 @@
      * @throws ConnectionFailedException The connection has died, and can no longer be used.
      */
     void send(Append append) throws ConnectionFailedException;
-    
+
+    /**
+     * flushes the pending events.
```

Comment:
Capital "F"

Reply:
Tom , advised to move this flush as WireComand in the file WirCommands.java file. All your review comments are indirectly taken care now.